src/client.js

```diff
@@ -106,8 +110,7 @@ Client.prototype.emit = function(event, data) {
 				if (target.chan.type === Chan.Type.LOBBY) {
 					chan = target.network.host;
 				}
-				userLog.write(
-					this.name,
+				this.userLog.write(
```

Comment:
Remove this entire log writing stuff out of this `emit` function and move log write calls where they're actually needed (pushMessage afaik)